server/app.js

```diff
@@ -16,7 +16,7 @@ const mainRouter = express.Router();
 mainRoutes(mainRouter, db);
 
 const app = express();
-app.use(cors({origin: 'http://localhost:3002'}));
```

Comment:
Move 'http://localhost:3000' to .env.development.local
https://create-react-app.dev/docs/adding-custom-environment-variables/
tip: env name must start with 'REACT_APP_'